IPython/core/display.py

```diff
@@ -961,20 +951,30 @@ def _jpegxy(data):
             # read another block
             idx += 2
 
-    h, w = struct.unpack('>HH', data[iSOF+5:iSOF+9])
-    return w, h
+    return struct.unpack('>HH', data[iSOF+5:iSOF+9])
```

Comment:
The code this is replacing swaps round the variables - it gets `h, w` and returns `w, h`. That seems like something it should keep.